web/src/components/common/Header.tsx

```diff
@@ -3,6 +3,7 @@ import { TopBar, Icon, Popover, Button } from '@equinor/eds-core-react'
 import { account_circle, grid_on, info_circle } from '@equinor/eds-icons'
 import { useContext, useRef, useState } from 'react'
 import { AuthContext } from 'react-oauth2-code-pkce'
+import { AUTH_DISABLED } from '../../App'
```

Comment:
Again, not sure if this constant should be exported from App.tsx. I find it hard to agree that Header is a common component now that it is dependent on another (non-common) component.

Reply:
Agreed. Header is not "common". It's only used one place.
But out of scope for this PR/issue

Reply:
The change was introduced in this PR, so if it's out of scope, well then changing the `AUTH_DISABLED` constant to a global is out of scope as well. The Header wasn't dependent on App.tsx until now.